farge

```diff
@@ -169,38 +173,36 @@ main() {
         # Write the color code in the center of the image
         convert "$FNAME" -gravity Center -pointsize 10 -fill "$TEXT_COLOR" -annotate 0 "$COLOR_CODE" "$FNAME"
     fi
-    
+
     # Save the color code to clipboard
     if [ "$WAYLAND_DISPLAY" ]; then
-        # We are removing the new line char and then sending it to the clipboard
-        echo "$COLOR_CODE" | tr -d "\n" | wl-copy
+        printf %s "$COLOR_CODE" | wl-copy
     else
-        echo "$COLOR_CODE" | tr -d "\n" | xclip -selection c
+        printf %s "$COLOR_CODE" | xclip -sel c
```

Comment:
You don't need `tr` if it's echo making the variable, just use `printf %s "$var"` instead. Or (not recommended) `echo -n "$var"`